### PR TITLE
Mobile madness

### PR DESCRIPTION
**Pull request by idosun**
- SDK upgrade (4.4.0_alpha)
- Introduce OkHttp for HTTP requests
- feedback dialog upon checkout crash
- Fullstory integration  

### Overview
Click checkout in the ToolStore and it produces a transaction with associated error.
Lifecycle methods from activities appear as breadcrumbs

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
+    <uses-permission android:name="android.permission.INTERNET"/>
```
Comment (thinkocapo) [inline app/src/main/AndroidManifest.xml:10]: it didn't have internet access by default?

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -62,9 +64,12 @@
             android:name="io.sentry.traces.sample-rate"
             android:value="1.0" />
 
+        <meta-data android:name="io.sentry.traces.activity.auto-finish.enable" android:value="false" />
```
Comment (thinkocapo) [inline app/src/main/AndroidManifest.xml:67]: why don't you want the activity's transaction to finish automatically?
https://docs.sentry.io/platforms/android/performance/included-instrumentation/#androids-activity-instrumentation

Are you manually finishing the transaction somewhere?

File: app/src/main/java/com/example/vu/android/MainActivity.java [hunk 3]
```diff
@@ -109,6 +104,17 @@ protected void onCreate(Bundle savedInstanceState) {
         });
     }
 
+    @Override
+    protected void onResume () {
+        super.onResume() ;
+
+        //We disabled tx auto finish for the 2nd activity
```
Comment (thinkocapo) [inline app/src/main/java/com/example/vu/android/MainActivity.java:111]: this answers my question

**Review by thinkocapo — COMMENTED** [review]
## Testing
https://sentry.io/organizations/testorg-az/performance/paranoid-android:ced787314fc74ef8bba6321251f81b8d/?project=1801383&query=transaction.duration%3A%3C15m+event.type%3Atransaction&showTransactions=recent&statsPeriod=7d&transaction=ToolStoreActivity&unselectedSeries=p100%28%29

https://sentry.io/organizations/testorg-az/performance/paranoid-android:61f5562b15e64958a96bceac2d1c9709/?project=1801383&query=transaction.duration%3A%3C15m+event.type%3Atransaction&showTransactions=recent&statsPeriod=7d&transaction=checkout+%5Bandroid%5D&unselectedSeries=p100%28%29

https://sentry.io/organizations/testorg-az/issues/2312469686/events/f76f134b67af454e8fd8416e4f85c646/?project=1801383

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (idosun) [reply app/src/main/AndroidManifest.xml:67]: If the activity is doing any async work then you have to finish the transaction manually...

**Pull request merged**